Mailnag/backends/__init__.py

```diff
@@ -41,7 +41,7 @@ def _str_to_folders(folders_str):
 
 
 def _folders_to_str(folders):
-	return json.dumps(folders)
+	return json.dumps(folders, ensure_ascii=False).encode('utf-8')
```

Comment:
Is it possible to decode the string returned by json.loads (line 37) to utf-8 as well, so the unicode-check in lolcal.py, line 122 can be dropped?